sailon/random_novelty_detector.py

```diff
@@ -10,10 +8,55 @@
 
 from smqtk_core import Configurable, Pluggable
 
+from .errors import RoundError
+from .novelty_interface import ONDAdapter
+
+
 logger = logging.getLogger(__name__)
 
 
-class RandomNoveltyDetector(Configurable, Pluggable):
+class RandomNoveltyDetectorAdapter(ONDAdapter):
```

Comment:
Could you move it to a different python file called `random_novelty_detector_adapter.py` or `rnd_adapter.py`?

Reply:
This makes sense, have moved the adapter class definition to its own file.